front/app/components/IdeaForm/index.tsx

```diff
@@ -842,7 +842,10 @@ class IdeaForm extends PureComponent<
             </FormElement>
           </StyledFormSection>
 
-          {(showPBBudget || showTopics || showLocation) && (
+          {(showPBBudget ||
+            showTopics ||
+            showLocation ||
+            showproposedBudget) && (
```

Comment:
`showproposedBudget` does not follow the camelCase naming conventions, and it is inconsistent with the other variable names. I know that it was already present in the code, but I suggest to change it to `showProposedBudget`.

Reply:
Ah, I didn't notice. Definitely - fixing it now :+1: